### ISSUE TITLE
Latest Sublime Text JavaScript Syntax Highlighting Irregularities

### ISSUE THREAD

**Issue by jonobr1** [thinkpixellab/flatland#129]
Hey there,

_Love_ the flatland theme, but noticed that when I updated to the latest Sublime Text 2 and 3 all my JS got way more orange. Here's what it used to look like:

<img width="1392" alt="screen shot 2016-02-12 at 9 43 05 am" src="https://cloud.githubusercontent.com/assets/220033/13015280/10c8ec96-d16d-11e5-87ef-734846a5e5ac.png">

And this is what it looks like now:

<img width="1392" alt="screen shot 2016-02-12 at 9 39 01 am" src="https://cloud.githubusercontent.com/assets/220033/13015287/158f397e-d16d-11e5-9449-425dd13a637a.png">

I'm not totally sure how themes work, but it looks like I'm not alone: https://forum.sublimetext.com/t/javascript-syntax-highlighting-changes-in-3103/17328

**Comment by bezet** [comment]
Indeed, the theme must be updated because last ST3 build introduced new syntax higlighting rules. I hope it won't take long since flatland is the best ST theme ever (EVER!) and now its js syntax higligthing is pretty useless - like almost everything is orange.

**Comment by jonobr1** [comment]
I'll look into this further. I'm sure it's just tweaking the Flatland Dark.tmTheme

**Comment by jonobr1** [comment]
wbond over on the Sublime Text Forum recommends updating themes like so:

````
<dict>
	<key>name</key>
	<string>JS Accessor</string>
	<key>scope</key>
	<string>keyword.operator.accessor.js</string>
	<key>settings</key>
	<dict>
		<key>foreground</key>
		<string>#F8F8F2</string>
	</dict>
</dict>
````

I'm going to fork and give this a try.

**Comment by sgerace** [comment]
@jonobr1 Any update on this? I've been holding off updating Sublime because of the breaking changes to the Flatland theme.

**Comment by nalinbhardwaj** [comment]
No update?
This issue has been open since 2 months now? :/

**Comment by nasitra** [comment]
Adding the following settings to "Flatland Dark.tmTheme", most of the highlighting looks previous one.

https://gist.github.com/nasitra/4549c3e7a5c13cf055a63ddc8c8f5872

**Comment by nalinbhardwaj** [comment]
Anything similar for C++ and other languages?
Why isn't the theme being updated itself?
Latest commit is from Jan 2015. Has development been stopped?
